collect_env.py

```diff
@@ -512,6 +515,16 @@ def is_xnnpack_available():
     else:
         return "N/A"
 
+def get_env_vars():
+    env_vars = ''
+    secret_terms=('secret', 'token', 'api', 'access', 'password')
+    for k, v in os.environ.items():
+        if any(term in k.lower() for term in secret_terms):
+            v = '***'
+        if k not in environment_variables:
+            v = '***'
+        env_vars = env_vars + "{}={}".format(k, v) + "\n"
```

Comment:
Instead of just overwriting the value, I would exclude the env var completely from the output if it's not a VLLM related env var.

Reply:
At beginning, it implements this way. Later [joerunde](https://github.com/joerunde) suggested `We could use v = *** or something like that to denote that the environment variable was set, but redacted here`. I think it makes sense to make it easier to debug.

Reply:
I see ... it makes me a little uncomfortable from a privacy perspective, but really anyone running this should be inspecting the output before they post it publicly.

There are also lots of env vars directly related to vLLM that won't be in vllm's list, like https://pytorch.org/docs/stable/torch_environment_variables.html ... so overall I'm convinced. Thanks for pointing me to the history on the topic.